cluster/src/dunit/scala/io/snappydata/externalstore/ColumnTableDUnitTest.scala

```diff
@@ -38,6 +38,19 @@ class ColumnTableDUnitTest(s: String) extends ClusterManagerTestBase(s) {
     startSparkJob()
   }
 
+  def testTableCreationWithHA(): Unit = {
+    val tableName = "TestTable"
+    val snc = SnappyContext(sc)
+    createTable(snc, tableName, Map("BUCKETS" -> "1", "PERSISTENT" -> "async"))
+    verifyTableData(snc , tableName)
```

Comment:
Ok. But where is data? I think it is empty. Isn't it? I meant put some data and then verify. Or am I missing something?